includes/Core/Util/Debug_Data.php

```diff
@@ -323,4 +324,26 @@ function ( Module_With_Debug_Fields $module ) {
 
 		return array_merge( array(), ...$fields_by_module );
 	}
+
+	/**
+	 * Gets the number of users with a Site Kit token.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	private function get_connected_user_count_field() {
+		$users = new \WP_User_Query(
+			array(
+				// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
+				'meta_key' => 'wp_googlesitekit_access_token',
```

Comment:
A few small details here:
- For the key name, please use the constant `OAuth_Client::OPTION_ACCESS_TOKEN` rather than the literal value
- Regarding the `wp_` – we can't assume this prefix because it varies by table prefix and network mode, so we have `User_Options::get_meta_key` to handle this for us 🙂 

See `\Google\Site_Kit\Core\Authentication\Has_Connected_Admins::query_connected_admins` for an example, only the `meta_key` here needs to change though.